bundles/org.openhab.binding.myq/README.md

```diff
@@ -32,7 +32,9 @@ Once an account has been added, garage doors and lamps will automatically be dis
 |---------------|---------------|------------------|--------------------------------------------------------|
 | status        | String        | garagedoor       | opening, closed, closing, stopped, transition, unknown |
 | rollershutter | Rollershutter | garagedoor       | UP, DOWN, 0%, 100%                                     |
-| switch        | Switch        | garagedoor, lamp | ON (open), OFF (closed)
+| closeError    | Switch        | garagedoor       | ON (has error), OFF (doesn't have error)               |
+| openError     | Switch        | garagedoor       | ON (has error), OFF (doesn't have error)               |
+| switch        | Switch        | garagedoor, lamp | ON (open), OFF (closed)                                
```

Comment:
```suggestion
| switch        | Switch        | garagedoor, lamp | ON (open), OFF (closed)                                |
```

Reply:
One last change, i think my copy/paste truncated the last `|` in my previous suggestion

Reply:
@digitaldan, thanks!

Reply:
@digitaldan I committed your suggestion